test/main/java/com/microsoft/store/partnercenter/TestConstants.java

```diff
@@ -0,0 +1,40 @@
+// -----------------------------------------------------------------------
+// <copyright file="TestConstants.java" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+package com.microsoft.store.partnercenter;
+
+public class TestConstants 
+{
+    /**
+     * The Active Directory authority endpoint address.
+     */
+    public static final String AadAuthorityEndpoint = "https://login.microsoftonline.com";
```

Comment:
The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_").

https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html